fpdf/html.py

```diff
@@ -664,11 +664,11 @@ def set_style(self, tag=None, enable=False):
     def set_text_color(self, r=None, g=0, b=0):
         self.pdf.set_text_color(r, g, b)
 
-    def put_link(self, txt):
+    def put_link(self, text):
```

Comment:
Shouldn't this method be decorated as well?
Or is it only ever used internally? In that case, how about `_put_link()` ?

Reply:
This method is is not part of `fpdf2` public API, so no, no need to decorate it.

Actually, none of the methods in `HTML2FPDF` are publicly callable by end users,
so I'm not sure that there is a need to make any of them start with an underscore.

Do you agree?

Reply:
That's partly a matter of style. The other part is that it causes those methods to appear in the autogenerated API documentation. There are ways to avoid that (as implemented eg. in drawing.py), which may or may not be worth the effort.

Reply:
I also don't think that using `drawing.force_nodocument()` is really worth the hassle...
I even have to admit that I do not quite understand the point of "hiding" those `render()` methods in the `drawing` module...

Anyway, I opened #904 to make explicit that the content of the `drawing` & `html` modules are internal APIs.